src/core/Akka/Actor/Scheduler/DedicatedThreadScheduler.cs

```diff
@@ -70,12 +81,16 @@ public DedicatedThreadScheduler(ActorSystem system)
                         }
                     }
                 }
+
+                // shutdown has been signaled
+               FireStopSignal();
```

Comment:
Signal to the caller that termination has completed, which allows the rest of the shutdown process to complete.